project-base/storefront/components/Blocks/Popup/LoginPopup.tsx

```diff
@@ -34,6 +34,15 @@ export const LoginPopup: FC<LoginPopupProps> = ({ defaultEmail, shouldOverwriteC
         ['/reset-password', '/registration'],
         url,
     );
+    const [socialNetworkLoginFacebook, socialNetworkLoginGoogle, socialNetworkLoginSeznam] =
+        getInternationalizedStaticUrls(
+            [
+                { url: '/social-network/login/:type', param: 'Facebook' },
+                { url: '/social-network/login/:type', param: 'Google' },
+                { url: '/social-network/login/:type', param: 'Seznam' },
+            ],
+            url,
+        );
```

Comment:
It would be nice to provide those data in SettingsQuery. Now you have to synchronize FE and BE separately. What if some project will want to have social logins only on some domains? Then you need to duplicate logic on both sides.

Reply:
I understand there will be a separate user story for that. @sspooky13 Could you please make sure that PO creates that user story?

Reply:
I don't have any idea how exactly I should do it 🤷‍♀️🤷‍♂️🤷 so I created user story myself and sent info to channel, I hope it's ok now